### DISCUSSION TITLE
Proposed DEI Initiative: Annual reporting on O3DECon speaker and attendee diversity

### DISCUSSION THREAD

**Discussion by AMZN-Liv** [o3de/community#71] [Diversity]
O3DECon could track gender and racial identity of speakers and attendees and publish them online, as an impact report. Example - [Grace Hopper Impact Report](https://indd.adobe.com/view/1b5adf4d-697a-4ad9-a306-db3dc55ce018). GDC also has a similar annual report that [surveys participants/registrants](https://gdconf.com/news/dont-forget-take-gdc-2021-state-industry-survey) of the conference, which can help O3DE/O3DF track and evaluate trends related to the open source community around major conference events. 

This would help the organization benchmark, set goals, and track improvements around outreach and participation to diverse communities within the project, specifically with regards to the conference. Data collected about the conferences can provide insight into places where processes and mechanisms need to be changed to encourage more diverse participation and representation.

**Comment by caniszczyk** [comment]
+1 FYI many LF foundations do an annual report that will include this information, see https://www.cncf.io/cncf-annual-report-2020/ and https://www.cncf.io/blog/2021/06/11/kubecon-cloudnativecon-europe-2021-virtual-conference-transparency-report/ as examples.

**Comment by debgiles** [comment]
Royal shared this request with me yesterday.  We will create a transparency report post-event for O3DECon every year (though the size/scope of the report will be a bit different from CNCF's, just given the difference in size/scope of the two events this year), and that report will include speaker and attendee demographics, along with lots of other summary information.  The information in that report can then be pulled from, should O3D Foundation choose to do an Annual Report in the future as well.
